lib/active_fedora/versionable.rb

```diff
@@ -10,7 +10,7 @@ module Versionable
     module ClassMethods
       def has_many_versions
         Deprecation.warn Versionable, "has_many_versions is deprecated and will be removed in ActiveFedora 11."
-        # self.versionable = true
+        self.versionable = true
```

Comment:
Add a comment here that this is just for marking datastreams with versions for migration by fedora-migrate.